controllers/openshift_controller.go

```diff
@@ -1239,30 +1272,25 @@ func (r *KataConfigOpenShiftReconciler) createExtensionMc(machinePool string) (b
 	var dummy bool
 
 	/* Create Machine Config object to enable sandboxed containers RHCOS extension */
-	mc := &mcfgv1.MachineConfig{}
-	err := r.Client.Get(context.TODO(), types.NamespacedName{Name: extension_mc_name}, mc)
-	if err != nil && (k8serrors.IsNotFound(err) || k8serrors.IsGone(err)) {
 
-		r.Log.Info("creating RHCOS extension MachineConfig")
-		mc, err = r.newMCForCR(machinePool)
-		if err != nil {
-			return dummy, err
-		}
+	if r.ExistingMc != nil {
+		r.Log.Info("Existing MachineConfig already exists")
```

Comment:
This log message sounds kind of tautological, was it meant to say "there is an existing MC already" or "using an existing MC" or something like that?